core/src/test/java/com/amplifyframework/analytics/BasicAnalyticsEventTest.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright 2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+package com.amplifyframework.analytics;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+public class BasicAnalyticsEventTest {
+
+    @Test
+    @SuppressWarnings("MagicNumber")
```

Comment:
Not part of this PR -- but just mentioning while I'm here --

I've been meaning to do a PR that adds `MagicNumber` to the `checkstyle-suppressions.xml` when found under `src/test` or `src/androidTest` (these trees tend to have lots of abitrarily arranged data values.)

Reply:
In the spirit of the annotations slightly irking me, I've tackled that in a commit in this pull. If you'd rather, I can split it out.